src/m365/spo/commands/customaction/customaction-get.spec.ts

```diff
@@ -392,6 +394,103 @@ describe(commands.CUSTOMACTION_GET, () => {
     }
   });
 
+  it('retrieves a user custom actions by clientSideComponentId', async () => {
+    sinon.stub(Cli, 'executeCommandWithOutput').callsFake(async (command): Promise<any> => {
```

Comment:
Rather than mocking command execution, let's mock here the actual requests. That will allow us to decouple tests from the actual implementation should we choose to take a different path for retrieving custom actions than by calling other command